src/pages/en/news/fabrication-for-beginners.mdx

```diff
@@ -0,0 +1,7 @@
+---
+image: /images/news/fabrication-for-beginners.jpeg
+target: https://www.instagram.com/p/CmO7AJ3Mqog/
+instagram: true
+title: School kids, veterans, and IDPs together learned digital fabrication for beginners.
+order: 1
```

Comment:
Macht es hier vielleicht eher Sinn, die Order zu reversen, bzw. vielleicht auch irgendwie in den Dateinamen zu schreiben?

Man möchte ja später easy neue hinzufügen, so muss man erst in jede File gehen und die Order anpassen, bevor man eine neue `order: 1` hinzufügen kann.

Reply:
Hatte nur den Content falsch übertragen. Wie du schon sagst: Höchste Ziffer sollte der neuste Content sein, damit man nicht alle alten Sachen um-nummerieren muss. Habe die Order in den Content-Files jetzt umgedreht.